### DISCUSSION TITLE
Search and read performance question

### DISCUSSION THREAD

**Discussion by jlegind** [minio/minio#14801] [Q&A] [answered]
Dear Minio users,

I need a fast image/file server for searching and retrieving among tens of millions of images serving the scientific community.
Can any of you tell me about the read performance of Minio across say 2 or 4 servers with SSD drives. Like a small cluster with decently spec'ed machines.

Best regards,
Kris

**Comment by harshavardhana** [comment, marked as answer]
Come over to our subscription we will help you get everything setup with architecture review https://min.io/pricing

**Reply by jlegind** [reply]
Not really answering my question.

**Reply by harshavardhana** [reply]
> Not really answering my question.

I did, what I meant is we do not provide performance and architecture advices without subscription.

If you wish to DIY please join https://slack.min.io
